shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/carmin/controller/CarminDatasetProcessingApiController.java

```diff
@@ -37,6 +40,11 @@ public class CarminDatasetProcessingApiController implements CarminDatasetProces
     @Autowired
     private ExecutionStatusMonitorService executionStatusMonitorService;
 
+    @Autowired
```

Comment:
Hi Alae, lines 43-47 could be completely removed if moved to annotation in CDProcApi.java